prefect_snowflake/database.py

```diff
@@ -8,6 +8,9 @@
 
 from prefect_snowflake.credentials import SnowflakeCredentials
 
+BEGIN_TRANSACTION_STATEMENT = "BEGIN"
```

Comment:
After looking at the [Snowflake docs](https://docs.snowflake.com/en/sql-reference/transactions.html#explicit-transactions), I think that this statement needs to be updated.
```suggestion
BEGIN_TRANSACTION_STATEMENT = "BEGIN TRANSACTION"
```